pkg/pillar/cmd/nodeagent/nodeagent.go

```diff
@@ -56,7 +56,7 @@ const (
 	minRebootDelay          uint32 = 30
 	maxDomainHaltTime       uint32 = 300
 	domainHaltWaitIncrement uint32 = 5
-	maxReadSize                    = 16384 // From files in /persist
+	maxReadSize                    = 65536 // From files in /persist
```

Comment:
Doesn't this affect other reads from /persist? I think we might meed a separate constant for maxSmartCtlSize.

Reply:
Good point, fixed.